index.html

```diff
@@ -57,16 +57,25 @@ <h2>Jobs & Projects</h2>
     <div class="row">
         <div class="col-sm-12 supporters">
             <h2>Supporters</h2>
-              <img src="images/supporters/creativecommons.svg" height="40px" width="auto" alt="Creative Commons Logo"/>
-              <img src="images/supporters/discourse.svg" height="50px" width="auto" alt="Discourse Logo"/>
-              <img src="images/supporters/mozilla.svg" height="50px" width="auto" alt="Mozilla Logo"/>
-              <img src="images/supporters/nextcloud.svg" height="70px" width="auto" alt="Nextcloud Logo"/>
-              <img src="images/supporters/opentechfund.svg" height="40px" width="auto" alt="OpenTechFund Logo"/>
-              <img src="images/supporters/phplist.svg" height="50px" width="auto" alt="PhpList Logo"/>
-              <img src="images/supporters/redhat.svg" height="50px" width="auto" alt="RedHat Logo"/>
-              <a title="Stickermule" href="httsp://stickermule.com/supports/opensourcedesign19-sponsorship" target="_blank" rel="noopener"><img src="images/supporters/stickermule.svg" height="40px" width="auto" alt="Stickermule Logo"/></a>
-              <img src="images/supporters/wikimedia-germany.svg" height="50px" width="auto" alt="Wikimedia Germany Logo"/>
-              <br>
+            <a title="creativecommons" href="https://creativecommons.org/">
```

Comment:
Hey!  It looks like the indents are a bit off.  All of these tags will need to be shifted 2 spaces forward.  Thanks for contributing!

Reply:
Only the img tags though, no? Because the a elements seem to be properly indented, same level as the h2 above it.